src/components/List.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import {Image, StyleSheet, Text, View} from 'react-native';
+import {Crypto} from './Interfaces';
+
+export const List = ({cryptos} /* :Crypto[] */) => {
+  let bull: boolean;
+
+  return (
+    <>
+      {cryptos.map((el: Crypto) => {
+        return (
+          <>
+            <View style={[styles.cryptoContainer, {padding: 25}]}>
+              <View style={styles.cryptoContainer}>
+                <Image style={styles.image} source={el.img} />
+                <View style={[{paddingLeft: 5}]}>
+                  <Text style={styles.name}> {el.name} </Text>
+                  <Text> {el.symbol} </Text>
+                </View>
+              </View>
+              <View style={styles.lineContainer}>
+                <View>
+                  <Text style={styles.name}> ${el.price} </Text>
```

Comment:
Since you're using name style on multiple components you should give it a more appropriate name. Also, use toFixed method on the price to limit the number of decimals.